src/Resources/scaffolds/6.0/bootstrapcss.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Bundle\MakerBundle\FileManager;
+
+return [
+    'description' => 'Add bootstrap css/js.',
+    'packages' => [
+        'symfony/twig-bundle' => 'all',
+        'symfony/webpack-encore-bundle' => 'all',
+    ],
+    'js_packages' => [
+        'bootstrap' => '^5.0.0',
+        '@popperjs/core' => '^2.0.0',
+    ],
+    'configure' => function (FileManager $files) {
+        // add bootstrap form theme
+        $files->manipulateYaml('config/packages/twig.yaml', function (array $data) {
+            $data['twig']['form_themes'] = ['bootstrap_5_layout.html.twig'];
+
+            return $data;
+        });
+
+        // add bootstrap to app.css
+        $files->dumpFile('assets/styles/app.css', "@import \"~bootstrap/dist/css/bootstrap.css\";\n");
```

Comment:
I agree in general, the reason I chose to override in this case is the default style from the recipe adds a sample background which makes the starter kit look crappier... what do you think? Another option is to include the file in the scaffold - this would trigger the confirmation (once this feature is added).